spec/src/main/asciidoc/core/Footnotes.adoc

```diff
@@ -17,7 +17,7 @@ footnote:a10234[This is to allow the container to swap out an instance’s state
 footnote:a10235[Any initialization methods defined for the bean by means of the `init-method` deployment descriptor element apply in addition to those defined by means of annotations.]   
 footnote:a10236[Any remove methods defined for the bean by means of the `remove-method` deployment descriptor element apply in addition to those defined by means of annotations.]   
 footnote:a10237[The concurrency management type `CONTAINER` may be specified for stateful session beans, but doing so has no impact on the semantics of concurrency management for such beans.]
-footnote:a10238[The `javax.ejb.ConcurrentAccessException` is a subclass of the `javax.ejb.EJBException`. If the business interface is a remote business interface that extends `java.rmi.Remote`, the client will receive the `java.rmi.RemoteException` instead.]
+footnote:a10238[The `jakarta.ejb.ConcurrentAccessException` is a subclass of the `jakarta.ejb.EJBException`. If the business interface is a remote business interface that extends `java.rmi.Remote`, the client will receive the `java.rmi.RemoteException` instead.]
```

Comment:
Changes to footnotes.adoc are not required since this is no longer referenced. I have removed footnotes.adoc in the other PR #121 .